packages/clay-form/src/SelectBox.tsx

```diff
@@ -154,6 +154,19 @@ const ClaySelectBox: React.FunctionComponent<IProps> = ({
 		Array.isArray(value) ? value : [value]
 	);
 
+	const selectedIndex = selectedIndexes.length === 1 && selectedIndexes[0];
+
+	const noItems = !items.length;
+
+	const noItemsSelected = !selectedIndexes.length;
+
+	const firstItemSelected =
+		selectedIndex === 0 || selectedIndexes.includes(0);
+
+	const lastItemSelected =
+		selectedIndex === items.length - 1 ||
+		selectedIndexes.includes(items.length - 1);
```

Comment:
I don't think you need the `selectedIndex` variable. Can't this just be simplified to this?

```suggestion
	const noItems = !items.length;
	const noItemsSelected = !selectedIndexes.length;
	const firstItemSelected = selectedIndexes.includes(0);
	const lastItemSelected = selectedIndexes.includes(items.length - 1);
```

Reply:
It can! @bryceosterhaus good job spotting that, because `selectedIndexes` is always an array, even with just one selected item, `selectedIndexes.includes()` applies always.